docs/DeveloperGuide.adoc

```diff
@@ -316,20 +316,108 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
-|`* *` |user |hide link:#private-contact-detail[private contact details] by default |minimize chance of someone else seeing them by accident
+|`* * *` |user |edit an entry |change its particulars (e.g. time, deadline)
 
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
+|`* * *` |user |delete an entry |remove listings that I no longer need
+
+|`* * *` |user |store a task without a due date and still get frequent reminders |remember to finish it
+
+|`* * *` |user |specify the start and end times for events |know when it takes place
+
+|`* * *` |user |search entries by name |retrieve specific entries more quickly
+
+|`* * *` |user |perform a search using keywords |find specific entries
+
+|`* * *` |user |store my entries in a folder managed by syncing softwares |ensure all my devices have the same list
+
+|`* * *` |user |list my entries based on starting times |view events scheduled according to specific times
+
+|`* * *` |user |archive entries to a 'completed tasks' folder |view tasks I have finished and restore them if needed
+
+|`* * *` |user |input the details (e.g. deadline, tags, time) of an entry in any order |add to the list without memorizing a specific format
+
+|`* * *` |user |automatically insert floating entries into free time-slots |remember to do tasks without deadlines
+
+|`* * *` |user |view the next n entries |to see upcoming events to attend/tasks to finish
+
+|`* * *` |user that attends events |add entries that have fixed start and end times |
+
+|`* * *` |user |adjust the start and end date of entries |update tasks when requirements change
+
+|`* *` |user |add a priority level to entries |filter them according to importance
+
+|`* *` |user |add recurring events |schedule daily/weekly/other cyclic tasks easily
+
+|`* *` |new user |have similar syntax for most commands |use them easily without confusion
+
+|`* *` |user |have a set of defaults for each field in a command|leave certain fields blank
+
+|`* *` |user |search entries by due-date/event-date |check my availability on a certain day
+
+|`* *` |user |search entries by added date |
+
+|`* *` |user |edit entries after searching for them |search and update in a one-shot approach
+
+|`* *` |user |have a daily view of tasks and events |view my workload for the day
+
+|`* *` |user |have a weekly view of tasks and events |view my workload for the week
+
+|`* *` |advanced user |undo my commands any number of times |to revert undesirable changes
+
+|`* *` |user |recover deleted entries |retrieve them in case of accidental deletions
+
+|`* *` |user with many projects |tag entries by assignment |distinguish tasks for similar assignments
+
+|`* *` |user with many projects |view entries by assignment |distinguish different assignments
+
+|`* *` |user |search for free days in a time period (e.g. next 1 month) |use free days to slot tasks
+
+|`* *` |user |launch and minimize the application |
```

Comment:
launch with a keyboard shortcut. 
missing the keyboard shortcut, right now it looks we want to be able to launch and minimize which of course we can